src/lib/registration/ui/base_system_registration_dialog.rb

```diff
@@ -358,17 +359,46 @@ def help_text
       #
       # @return [Boolean] true when skipping has been confirmed
       def show_skipping_warning
+        media_name = ProductFeatures.GetStringFeature(
+          "globals",
+          "full_system_media_name"
+        )
+        download_url = ProductFeatures.GetStringFeature(
+          "globals",
+          "full_system_download_url"
+        )
+
+        warning = _("Without registration, update channels will not be\n" \
+          "configured. This disables updates and security fixes.")
+
         # Popup question: confirm skipping the registration
         # TRANSLATORS:
         # %{media_name} is the media name (e.g. SLE-15-Packages),
         # %{download_url} is an URL link (e.g. https://download.suse.com)
-        warning = _("Without registration, update channels will not be\n" \
-          "configured. This disables updates and security fixes.\n\n" \
-          "A full system can be installed using the\n" \
-          "%{media_name} media from %{download_url}.\n" \
-          "Without these media only a minimum system is available\n" \
-          "in this installation.") %
-          { media_name: "SLE-15-Packages", download_url: "https://download.suse.com" }
+        if media_name && !media_name.empty? &&
+            download_url && !download_url.empty?
+          warning += "\n\n" +
+            _("A full system can be installed using the\n" \
+            "%{media_name} media from %{download_url}.") %
+            { media_name: media_name, download_url: download_url }
+        elsif media_name && !media_name.empty?
```

Comment:
I'm thinking if handling of the other cases is needed for the future. It feels a bit like over engineering...

I think the additional medium will always have some name (even if a stupid one like `DVD2`), you need to be able to somehow tell the customers to use an additional medium and that's hard without a name. :grin: 

Maybe we could simply require defining both and drop these extra conditions. On the other hand this extra code is pretty small and straightforward so I'll leave the decision up to you. :smiley:

Reply:
I even do not have a testcase for that corner cases :-) --> removed.